assets/js/modules/books.js

```diff
@@ -0,0 +1,99 @@
+class Books {
+    static books = [];
+
+    static form;
+
+    static bookTitle;
+
+    static bookAuthor;
+
+    constructor(title, author) {
+      this.title = title;
+      this.author = author;
+    }
+
+    static initialize = () => {
+      Books.form = document.querySelector('form');
+      Books.bookTitle = document.querySelector('.title');
+      Books.bookAuthor = document.querySelector('.author');
+
+      const inv = new Books();
+      inv.submitForm();
+
+      window.addEventListener('DOMContentLoaded', () => {
+        if (localStorage.getItem('books')) {
+          const allBooks = JSON.parse(localStorage.getItem('books'));
+          inv.displayBooks(allBooks);
+        }
+      });
+    }
+
+    // eslint-disable-next-line class-methods-use-this
+     addBooks = () => {
+       const title = Books.bookTitle.value;
+       const author = Books.bookAuthor.value;
+       const bookDetails = new Books(title, author);
+
+       if (localStorage.getItem('books')) {
+         Books.books = JSON.parse(localStorage.getItem('books'));
+       }
+       Books.books.push(bookDetails);
+       localStorage.setItem('books', JSON.stringify(Books.books));
+     }
+
+     // eslint-disable-next-line class-methods-use-this
```

Comment:
- [ ] Great job working on this milestone. You should not disable eslint rules, but rather fix the errors raised. The linters are there to make us write robust codes. Kindly remove the disable rule.